src/saveload/afterload.cpp

```diff
@@ -3121,6 +3121,8 @@ bool AfterLoadGame()
 		}
 	}
 
+	if (IsSavegameVersionBefore(SLV_VEHICLE_CARGOTILES)) _settings_game.economy.idle_loading_ct_penalty = 2;
```

Comment:
This is just the default value from settings.ini, maybe this conversion isn't strictly necessary?

Reply:
Would've thought not necessary, yes

Reply:
Based on experience from a different patch, without this the converted save will take the current newgame setting. I think I will slightly prefer removing this line, so the player's newgame preference is used.